libs/libskynet/src/skylinkBitfield.ts

```diff
@@ -0,0 +1,166 @@
+// parseSkylinkBitfield parses a skylink bitfield and returns the corresponding
+// version, offset, and fetchSize.
+function parseSkylinkBitfield(skylink: Uint8Array): [bigint, bigint, bigint, string | null] {
```

Comment:
convention was decided after a lot of these files were created so it will be enforced inconsistently. We'll try to catch all the inconsistencies during code review.